docs/hu/common/core/coffee-jpa.adoc

```diff
@@ -57,473 +57,10 @@ public class TransactionHelperExample {
-----
-| Átkonvertálódik *String* típusra, majd a *PreparedStatement.setString()* metódusa használatával kerül beállításra.
-
-a|
-[source,java]
-----
-@Column(name = "CHAR_WRAPPER")
-private Character charWrapper;
-----
-| Átkonvertálódik *String* típusra, majd a *PreparedStatement.setString()* metódusa használatával kerül beállításra.
-
-|===
-
-IMPORTANT: A táblázatban nem szereplő típusok beállítását a *JDBC driver*-re bízzuk.
-
-=== Adat típusú BasicType kezelése [[DataBasicType]]
-
-Az *SqlTypes.BLOB*, *SqlTypes.VARBINARY* és *SqlTypes.LONGVARBINARY* jdbc típuskóddal rendelkező *BasicType* típusok a *BatchService* által a következőképpen kerülnek kezelésre:
-
-[%header,cols=2]
-|===
-| Java kód
-| BatchService által beillesztett érték
-
-a|
-[source,java]
-----
-@Lob
-@Column(name = "DEFAULT_BLOB")
-private Blob defaultBlob;
-----
-| Átkonvertálódik *InputStream* típusra, majd a *PreparedStatement.setBinaryStream()* metódusa használatával kerül beállításra.
-
-a|
-[source,java]
-----
-@Column(name = "PRIMITIVE_BYTE_ARRAY")
-private byte[] primitiveByteArray;
-----
-| Közvetlenül, átalakítás nélkül beállítható a *PreparedStatement.setBytes()* metódusa használatával.
-
-a|
-[source,java]
-----
-@Column(name = "WRAPPER_BYTE_ARRAY")
-private Byte[] wrapperByteArray;
-----
-a| Ha a legacy array handling engedélyezve van:
-
-* Átkonvertálódik primitív *byte[]* típusra, majd a *PreparedStatement.setBytes()* metódusa használatával kerül beállításra.
-
-Ha nincs engedélyezve a legacy array handling:
-
-* A használt *JDBC driver*-re bízzuk a feloldást.
-
-IMPORTANT: Amennyiben lehetséges, byte[] használata ajánlott helyette.
-
-a|
-[source,java]
-----
-@Lob
-@Column(name = "LOB_PRIMITIVE_BYTE_ARRAY")
-private byte[] lobPrimitiveByteArray;
-----
-| Közvetlenül, átalakítás nélkül beállítható a *PreparedStatement.setBytes()* metódusa használatával.
-
-a|
-[source,java]
-----
-@Lob
-@Column(name = "LOB_WRAPPER_BYTE_ARRAY")
-private Byte[] lobWrapperByteArray;
-----
-a| Ha a legacy array handling engedélyezve van:
-
-* Átkonvertálódik primitív *byte[]* típusra, majd a *PreparedStatement.setBytes()* metódusa használatával kerül beállításra.
-
-Ha nincs engedélyezve a legacy array handling:
-
-* A használt *JDBC driver*-re bízzuk a feloldást.
-
-IMPORTANT: Amennyiben lehetséges, byte[] használata ajánlott helyette.
-
-|===
-
-IMPORTANT: A táblázatban nem szereplő típusok beállítását a *JDBC driver*-re bízzuk.
-
-[NOTE]
-====
-A legacy array handling engedélyezéséhez a `persistence.xml` bővítése szükséges a következővel: `<property name="hibernate.type.wrapper_array_handling" value="legacy"/>`. +
-Bővebben: https://github.com/hibernate/hibernate-orm/blob/6.2/migration-guide.adoc#bytecharacter-mapping-changes[hibernate 6.2 migration guide]
-====
+Az *IJpaBatchService* a tömeges adatbázis műveletek (insert, update, delete) csoportosan való végrehajtására szolgál.
+Hibernate alapú implementációja a `frappee-hibernate` modulban található.
```

Comment:
Egy link menjen rá pls hol található ez. Illetve ide majd kellene egy minta hogy lehet ezt használni, mit kell pl dependency-ben hozzáadni. (Tudom még nincs kész a másik oldal, így ez vagy egy issue, vagy várjuk meg vele a frappee-t)

Reply:
A frappee-hoz meg kelleni fog a coffee és így körbe is értünk, csinálok rá egy issuet.

Reply:
https://github.com/i-Cell-Mobilsoft-Open-Source/coffee/issues/662 -ben lesz megoldva.